SWLOR.Game.Server/Feature/RecipeDefinition/SmitheryRecipeDefinition/ArmorRecipes.cs

```diff
@@ -54,6 +54,54 @@ private void Tier1()
                 .EnhancementSlots(RecipeEnhancementType.Armor, 1)
                 .Component("lth_ruined", 4)
                 .Component("fiberp_ruined", 2);
+
+            // Advent Tunic
+            _builder.Create(RecipeType.AdventTunic, SkillType.Smithery)
```

Comment:
I forgot to tell you about a step. The RecipeType enumeration needs entries too. Make sure the numbers you assign there are unique. There are sections for each craft's recipes.